runtime/parachains/src/configuration.rs

```diff
@@ -784,6 +789,7 @@ pub mod pallet {
 		))]
 		pub fn set_max_upward_message_size(origin: OriginFor<T>, new: u32) -> DispatchResult {
 			ensure_root(origin)?;
+			ensure!(new <= T::MaxUpwardMessageSizeBound::get(), Error::<T>::InvalidNewValue);
```

Comment:
Perhaps, this should also be reflected in `check_consistency`?

Reply:
@pepyakin As I see it, I have to change the generic parameter of `HostConfiguration` from `Config::BlockNumber` to `Config` make get access to `MaxUpwardMessageSizeBound`. Is this ok?

Reply:
Hey @pepyakin When I moved the checks to `check_consistency`, running any chain fails because in `node/service/src/chain_spec.rs` and `bridges/bin/rialto/node/src/chain_spec.rs` the max-size is set to `1024*1024` and that is way higher than the 50KiB you proposed. What do you suggest to solve this?

Reply:
Ah, good to hear that `check_consistency` gets its job done. I say we decrease those numbers to 50 KiB.

Reply:
Hey @pepyakin or @kianenigma I'm stuck here with an issue and maybe you can help me. ATM the `service_overweight` bench fails bc. I'm unable to create an overweight message. When I limit the message size to 50KiB I won't reach the current weight limit to make it overweight.

Ideally one can create an arbitrary-weighted call by using `fill_block( req_weight / max_weight )` I tried it but it seems the execution of `fill_block` in `XcmExecutor::execute_xcm` does not consume any weight at all and so will not issue a `XcmError::WeightLimitReached`.

My idea was to create a `pallet_utilities::batch` with as many `remark` calls to reach overweight, but I ran into compiler issues with `T::Config` vs. `pallet_utilities::Config`. Do you have a better idea how to create a heavyweight call?
If the later approach is ok to you, I'd try to fix the issues (or ask more specific questions).